construct_btree_from_inorder_postorder.py

```diff
@@ -0,0 +1,33 @@
+"""
+// Time Complexity : O(n)
```

Comment:
Shouldn't time be O(n^2)? index is O(n) operation and you are having that on n elements.

Reply:
Oh, I meant to use the dict that I created to get the index. WIll update.